libs/platform/src/lib/components/wizard-generator/components/dialog-wizard-generator/dialog-wizard-generator.component.scss

```diff
@@ -0,0 +1,46 @@
+.fdp-wizard-dialog {
+
+    .fd-message-strip {
+        margin-bottom: 12px !important;
+    }
+
+    .fd-dialog {
+        &__header {
+            .fd-bar {
+                box-shadow: none;
+            }
+        }
+    }
+
+    .fd-title {
+        &--h3 {
+            margin-bottom: 12px !important;
+        }
+
+        &--h2 {
+            padding: 16px 0 8px 16px;
+        }
+    }
+
+    .fd-wizard {
+        &__content {
+            padding-left: 32px;
+            padding-right: 32px;
+        }
+
+        &__progress-bar {
+            padding-left: 32px;
+            padding-right: 32px;
+        }
+    }
```

Comment:
there are modifier classes for the responsive paddings:
<img width="1182" alt="Screen Shot 2021-06-28 at 9 19 32 AM" src="https://user-images.githubusercontent.com/39598672/123643083-0623d080-d7f2-11eb-84ce-be5bd3ca6647.png">